### DISCUSSION TITLE
有事务之类的机制吗？

### DISCUSSION THREAD

**Discussion by GUANchenchen023** [fireboomio/product-manual#57] [Q&A]
有事务之类的机制吗？

**Comment by GUANchenchen023** [comment]
并行，暂时不支持事务，有计划支持。

**Reply by AnsonCode** [reply]
Fireboom本身不支持事务，可以用钩子SDK实现事务支持，例如 [graphql钩子](https://ansons-organization.gitbook.io/product-manual/jin-jie-gou-zi-ji-zhi/graphql-gou-zi) 。
